community/cypher/cypher/src/test/java/org/neo4j/cypher/internal/javacompat/NotificationAcceptanceTest.java

```diff
@@ -90,15 +89,15 @@ public void shouldNotifyWhenUsingCypher3_1ForTheRulePlannerWhenCypherVersionIsTh
     @Test
     public void shouldWarnWhenRequestingCompiledRuntimeOnUnsupportedQuery()
     {
-        Stream.of( "CYPHER 3.1", "CYPHER 3.4" ).forEach(
+        Stream.of( "CYPHER 3.1", "CYPHER 3.5" ).forEach(
```

Comment:
There is a lot of those things in this file. Might be nice to do something like the following to save some work next time?
```
Stream.of( "CYPHER x.y", ... , CURRENT_VERSION )
```